my_fastapi_project/curl_api.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+curl http://127.0.0.1:8000/api/v1/hello
```

Comment:
Might want to add a new line to the end of your files to avoid those annoying github warnings.